### DISCUSSION TITLE
Migrating 169 bookmarks folders from maps.me into Organic Maps

### DISCUSSION THREAD

**Discussion by palzkillm** [organicmaps/organicmaps#2876] [Q&A] [answered]
Hello,

I'm moving away from maps.me, the newest version is no good. I'd like to take my Bookmarks with me. Unfortunately, I have 169 individual bookmarks folders with usually no more than five bookmarks in them - so a lot of files.

I know I can manually export each file from maps.me, and then open it in Organic Maps, but I was hoping there was a faster way to migrate.

I tried copying the bookmarks folder from maps.me into Organic Maps using itunes, but the files don't seem to get loaded by the App.

Any ideas for a faster migration?
Marco

P.S.: THANKS for bringing Organic Maps into our lives! Will soon start contributing!

**Comment by GhostInTheKayak** [comment]
I had the same problem with transferring a lot of groups so I could kill off the old app.

In the end I just set aside an hour or so to do the transfer directly between the apps.
I did it on my iPad with the two apps side-by-side, so it was unexpectedly straightforward.

**Comment by biodranik** [comment, marked as answer]
1. You can build and use a command-line tool from our repository called `kmb_to_kml`
2. We can add some additional kmb reading code for manually copied folders, but it will take a lot of time.
3. You can send me an archive of your bookmarks (alex аt organicmaps.app), so I can quickly convert them to kml.
